README.md

```diff
@@ -52,6 +52,14 @@ The Python interpreter to use for AWS SAM CLI. Defaults to `python` on Windows,
 
 You can use [`actions/setup-python`](https://github.com/actions/setup-python) to automatically set up Python.
 
+### `installer`
```

Comment:
Is this named clearly enough?

Reply:
We don't currently have consistent language around installers in documentation, so I didn't use something more specific (e.g. `native-installer`, `self-contained-installer`, etc.). And since it's always used under `with`, `with` `installer` kind of makes sense IMO (it's more of a regular installer rather than a Python package), but taking suggestions.

Reply:
nit: it is not super clear this is a boolean. how about `use_installer`

Reply:
With use installer? The wording is a little awkward.

Reply:
🤔 let me check other official gh action

Reply:
```
steps:
  - uses: actions/checkout@v3
  - uses: actions/setup-go@v3
    with:
      go-version: '1.14'
      check-latest: true
  - run: go run hello.go
```

yeah it is a verb. https://github.com/actions/setup-go

Reply:
Not here:

- https://github.com/ruby/setup-ruby/blob/92aece5fc9c784ab66851c1e702b1bd5885a51f2/action.yml#L24-L26
- https://github.com/actions/setup-node/blob/7c29869aec4da703a571b27bcd84d4f15af0b56e/action.yml#L5-L7

Reply:
I'll change to `use-installer`, both look fine to me. With next major version we'll most likely get rid of it (and `python`) anyway so not a big deal.

Reply:
Always cache also indicate Boolean. The other one is … meh